### PR TITLE
Add Starlette

### PR DESCRIPTION
**Pull request by tomchristie**
Add [Starlette](https://github.com/encode/starlette) to the Python frameworks.

### CODE REVIEW

File: frameworks/Python/starlette/app.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+
+@asgi_application
+def json_serialization(request):
+    return UJSONResponse({'message': 'Hello, world!'})
+
+
+@asgi_application
+async def single_database_query(request):
+    row_id = randint(1, 10000)
```
Comment (michaelhixson) [inline frameworks/Python/starlette/app.py]: Not that it matters a whole lot, but the highest number this generates is 9999, correct?  If so, I think this should be `randint(1, 10001)`.  There is a record with id=10000 in the database.

Comment (tomchristie) [reply frameworks/Python/starlette/app.py]: Nope, it’s 1-10000 inclusive.

Python’s docs for randint are: ‘Return a random integer N such that a <= N <= b’

(Unlike Pythons list slices)

Comment (michaelhixson) [reply frameworks/Python/starlette/app.py]: Oh my bad.  I was looking at the docs for numpy.random.randint.

Comment (tomchristie) [reply frameworks/Python/starlette/app.py]: Thanks for the review @michaelhixson - Have a fab weekend! ✨

**Pull request merged**